src/components/description-list/_description-list.scss

```diff
@@ -1,6 +1,14 @@
 .ons-description-list {
     &__items {
         margin: 0 0 2rem;
+
+        &.ons-description-list--horizontal {
+            @media (width >= 980px) {
```

Comment:
I dont think we should be using custom breakpoints here we should use the large breakpoint instead which is also set to 980px

You would do something like this for example:
```
@include mq(l) {
    margin-bottom: 1rem;
    margin-top: 1rem;
}
```